test/powershell/Modules/PSDiagnostics/PSDiagnostics.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe -Name "PSDiagnostics cmdlets tests" -Tag "CI","RequireAdminOnWindows" {
+    Context -Name "Test for Enable-PSTrace cmdlet" {
+        it -Name "Should enable Analytic logs for Microsoft-Windows-PowerShell." -Skip:(-not $IsWindows) {
+            Enable-PSTrace -Force
+
+            [XML]$WevtUtilOutput = & wevtutil gl Microsoft-Windows-PowerShell/Analytic /f:xml
+
+            $WevtUtilOutput.Channel.enabled | Should -Be 'true'
+        }
+    }
+
+    Context -Name "Test for Disable-PSTrace cmdlet" {
+        it -Name "Should disable Analytic logs for Microsoft-Windows-PowerShell." -Skip:(-not $IsWindows) {
+            Disable-PSTrace
+
+            [XML]$WevtUtilOutput = & wevtutil gl Microsoft-Windows-PowerShell/Analytic /f:xml
+
+            $WevtUtilOutput.Channel.enabled | Should -Be 'false'
+        }
+    }
+
+    Context -Name "Test for Get-LogProperties cmdlet" {
+        it -Name "Should show properties of Admin logs for 'Microsoft-Windows-PowerShell'." -Skip:(-not $IsWindows) {
+            [XML]$WevtUtilOutput = wevtutil gl Microsoft-Windows-PowerShell/Admin /f:xml
+
+            $LogProperty         = Get-LogProperties -Name Microsoft-Windows-PowerShell/Admin
+
+            $LogProperty.Name       | Should -Be $WevtUtilOutput.channel.Name
+            $LogProperty.Enabled    | Should -Be $WevtUtilOutput.channel.Enabled
+            $LogProperty.Retention  | Should -Be $WevtUtilOutput.channel.Logging.Retention
+            $LogProperty.AutoBackup | Should -Be $WevtUtilOutput.channel.Logging.AutoBackup
+            $LogProperty.MaxLogSize | Should -Be $WevtUtilOutput.channel.Logging.MaxSize
+        }
+    }
+
+    Context -Name "Test for Set-LogProperties cmdlet" {
+        BeforeAll  {
+            $LogType = 'Analytic'
+            if($IsWindows){
```

Comment:
```suggestion
            if ($IsWindows) {
```

Reply:
I think we use this convention here only for .cs.
@iSazonov Can you confirm on this ?

Reply:
Although CodeFactor passes and I'd prefer spaces that CodeFactor passes too. Definitely the entire file should be in the same style.

Reply:
Updated